packages/components/src/DataTable/DataTable.mdx

```diff
@@ -707,6 +711,70 @@ import { DataTable } from "@jobber/components/DataTable";
   />
 </Playground>
 
+## Table empty state
+
+<Playground>
+  <DataTable
+    height={400}
+    pagination={{ manualPagination: false, itemsPerPage: [10, 20, 30] }}
+    sorting={{ manualSorting: false }}
+    data={[]}
+    columns={[
+      {
+        id: "name",
+        accessorKey: "name",
+        header: "Name",
+        footer: "Totals",
+      },
+      {
+        id: "points",
+        accessorKey: "points",
+        header: () => (
+          <Typography align="end" fontWeight="bold">
+            Points
+          </Typography>
+        ),
+        footer: () => (
+          <Typography align="end" fontWeight="bold">
+            $0.00
+          </Typography>
+        ),
+        cell: info => <Typography align="end">{info.getValue()}</Typography>,
+      },
+      {
+        id: "chance",
+        accessorKey: "chance",
+        header: () => (
+          <Typography align="end" fontWeight="bold">
+            Chance (%)
+          </Typography>
+        ),
+        cell: info => <Typography align="end">{info.getValue()}</Typography>,
+      },
+      {
+        id: "power",
+        accessorKey: "power",
+        header: () => (
+          <Typography align="end" fontWeight="bold">
+            Power
+          </Typography>
+        ),
+        footer: () => (
+          <Typography align="end" fontWeight="bold">
+            $0.00
+          </Typography>
+        ),
+        cell: info => <Typography align="end">{info.getValue()}</Typography>,
+      },
+    ]}
+  >
+    <div className={classNames(styles.emptyMessage)}>
+      <Heading level={5}>No items found</Heading>
+      <Text>Update your search or filter selection</Text>
+    </div>
```

Comment:
Based on the changes you made, I believe this would now be captured as a React node and provided as the `children` argument to the above `<DataTable>` element.

Reply:
I believe children is a reserved word and are typically passed between the opening and closing tags of the component no? 🤔